elide-datastore/elide-datastore-jpa/src/test/java/com/yahoo/elide/datastores/jpa/LegacyNPlusOneIT.java

```diff
@@ -171,7 +171,7 @@ public void testSingleElementSubcollectionWithFilter() {
                 .statusCode(HttpStatus.SC_OK);
 
         verifyLoggingStatements(
-                "SELECT example_Book FROM example.Book AS example_Book WHERE example_Book.id IN (:XXX)",
+        //      "SELECT example_Book FROM example.Book AS example_Book WHERE example_Book.id IN (:XXX)",
```

Comment:
If we are not leveraging JPQL, we still probably want to log something here when we do a em.find.

Reply:
Made the changes to log.